overlay/libhybris/default.nix

```diff
@@ -35,6 +37,11 @@ stdenv.mkDerivation {
       --replace "/usr/bin/file" "${file}/bin/file"
   '';
 
+  NIX_LDFLAGS = [
+    # For libsupc++.a
+    "-L${stdenv.cc.cc.out}/${libPrefix}/lib/"
```

Comment:
Yes, this'll double-up to `//`, but that's basically a no-op in directory traversal.